content/zh/blogs/2024/docs-localized.md

```diff
@@ -0,0 +1,23 @@
+---
+title: OpenTelemetry 网站支持多语言！
+linkTitle: 多语言网站
+date: 2024-08-20
+author: >-
+  [Severin Neumann](https://github.com/svrnm) (Cisco)
+issue: 4863
+sig: Comms
+default_lang_commit: aca2e07f0d3a6
+---
+
+我们很高兴地宣布，OpenTelemetry 网站现已提供多种语言版本！本地化团队已经开始将网站页面翻译成 [中文](/zh), [日语](/ja), [葡萄牙语](/pt), and [西班牙语](/es)
```

Comment:
","  ==> "、"

Reply:
done

Reply:
Need not reply each comment. You can only click `Resolve conversation` if you have applied the suggestions.